src/main/java/competition/subsystems/arm/commands/ControlArmsWithJoyStickCommand.java

```diff
@@ -0,0 +1,42 @@
+package competition.subsystems.arm.commands;
+
+import competition.operator_interface.OperatorInterface;
+import competition.subsystems.arm.LowerArmSubsystem;
+import competition.subsystems.arm.UpperArmSubsystem;
+import xbot.common.command.BaseCommand;
+import xbot.common.math.MathUtils;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+@Singleton
+public class ControlArmsWithJoyStickCommand extends BaseCommand {
+    private final OperatorInterface oi;
+    private final LowerArmSubsystem lowerArmSubsystem;
+    private final UpperArmSubsystem upperArmSubsystem;
+    @Inject
+    public ControlArmsWithJoyStickCommand(OperatorInterface oi, LowerArmSubsystem lowerArmSubsystem, UpperArmSubsystem upperArmSubsystem ){
+        this.oi = oi;
+        this.lowerArmSubsystem = lowerArmSubsystem;
+        this.upperArmSubsystem = upperArmSubsystem;
+
+    }
+
+    public void initialize(){
+
```

Comment:
add this code:

`log.info("Initializing!");`. 

This lets us follow what the robot did in the logs after a match.

Reply:
Code was added